pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -77,8 +77,8 @@ users:
 	flags.StringVar(&options.PublicAPIServerURL, "public-master", "", "The API public facing server's URL (if applicable).")
 	flags.StringVar(&options.APIServerCAFile, "certificate-authority", "openshift.local.certificates/ca/cert.crt", "Path to the API server's CA file.")
```

Comment:
No.  At least not yet.  That is where generate the master CA by default, so most of the time that's what you'd want to use.  If we decide to offer different sorts of default locations, then yes.